pkg/openapiclient/local_schemas.go

```diff
@@ -48,7 +51,10 @@ func (k *localSchemasClient) Paths() (map[string]openapi.GroupVersion, error) {
 			res[path] = groupversion.NewForFile(k.fs, filepath.Join(groupPath, v.Name()))
 		}
 	}
-	coregroup, _ := utils.ReadDir(k.fs, filepath.Join(k.dir, "api"))
+	coregroup, err := utils.ReadDir(k.fs, filepath.Join(k.dir, "api"))
```

Comment:
The `api` folder is allowed to not exist, we should return an error only if the folder exists AND there was an error reading it.